### PR TITLE
Removed troubleshooting from topology. Added to bastion docs.

### PR DESCRIPTION
**Pull request by evildandelions**
Closes https://github.com/kubernetes/kops/issues/1371

CC @kris-nova

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1490)

### CODE REVIEW

**Comment by kris-nova** [comment]
@k8s-bot ok to test

File: docs/bastion.md [hunk 1]
```diff
@@ -68,6 +68,29 @@ spec:
...
+### Using SSH agent to access your bastion
+
+Verify your local agent is configured correctly
+
+```
+$ ssh-add -L
+ssh-rsa <PUBLIC_RSA_HASH> /Users/kris/.ssh/id_rsa
+```
+
+If that command returns no results, add the agent to `ssh-agent`
```
Comment (justinsb) [inline docs/bastion.md]: (Pre-existing typo): should be "add the key to `ssh-agent`" or something like that I think

File: docs/bastion.md [hunk 2]
```diff
@@ -68,6 +68,29 @@ spec:
...
+
+```
+ssh-add ~/.ssh/id_rsa
+```
+
+Check the agent is now added using `ssh-add -L`
```
Comment (justinsb) [inline docs/bastion.md]: "key" again I think

**Comment by justinsb** [comment]
Thanks @evildandelions !

**Comment by evildandelions** [comment]
@justinsb Good catch- it's fixed now.

**Comment by justinsb** [comment]
LGTM - thanks!

**Pull request merged**